graphula-core/src/Graphula.hs

```diff
@@ -445,7 +464,7 @@ attemptsToInsertWith attempts source
 
 -- | For entities that only have singular 'Dependencies'. It uses data instead
 -- of newtype to match laziness of builtin tuples.
-data Only a = Only { fromOnly :: a }
+newtype Only a = Only { fromOnly :: a }
```

Comment:
@pbrisbin Comment is out of date now - this was using `data` on purpose to match builtin tuples, but it probably has limited utility